ioctl/util/util.go

```diff
@@ -149,6 +149,7 @@ func ReadSecretFromStdin() (string, error) {
 			case <-routineTerminate:
 				return
 			default:
+				return
```

Comment:
why add `return` here?

Reply:
Staticcheck warns: "ioctl\util\util.go:151:4: should not have an empty default case in a for+select loop; the loop will spin (SA5004)"